src/views/Item/ItemImage.js

```diff
@@ -10,10 +8,7 @@ import Image from '../../elements/Image'
  * An item can contain an image
  * */
```

Comment:
While we're at it, there are a number of doc blocks with `* */` as a closing.  This should just be `*/`.  The copy pasta there has been killing me a little :)